pkg/networkservice/chains/forwarder/server.go

```diff
@@ -73,23 +72,27 @@ type xconnectNSServer struct {
 }
 
 // NewServer - returns an implementation of the xconnectns network service
-func NewServer(ctx context.Context, name string, authzServer networkservice.NetworkServiceServer, tokenGenerator token.GeneratorFunc, clientURL *url.URL, vppConn Connection, tunnelIP net.IP, tunnelPort uint16, dialTimeout time.Duration, domain2Device map[string]string, clientDialOptions ...grpc.DialOption) endpoint.Endpoint {
+func NewServer(ctx context.Context, name string, authzServer networkservice.NetworkServiceServer, tokenGenerator token.GeneratorFunc, clientURL *url.URL, vppConn Connection, tunnelIP net.IP, dialTimeout time.Duration, domain2Device map[string]string, options ...Option) endpoint.Endpoint {
```

Comment:
Can we add follow options?

1. name (see as we're doing with names for clients and nsmgrs)
2. authzServer
3. clientURL
4. dialTimeout
5. domain2Device
```suggestion
func NewServer(ctx, tokenGenerator token.GeneratorFunc, clientURL *url.URL, vppConn Connection, tunnelIP net.IP, options ...Option) endpoint.Endpoint {
```

Reply:
Sure. Done